Makefile

```diff
@@ -176,6 +182,33 @@ STAGE2_UTIL_EXECUTABLES := \
 	runghc \
 	unlit
 
+STAGE2_TARGET_LIBS := \
```

Comment:
This is for the CROSS compiler, right? Can we rename it to STAGE2_CROSS_TARET_LIBS then?